jetson/power_logging/model/benchmark.py

```diff
@@ -35,18 +37,22 @@ def __init__(self, enable_timing = True):
             self.event = None 
 
     def record(self):
+        self.time_stamp = time.time()
```

Comment:
I still think this function is incorrect and should be replaced by either `time.perf_counter` or `time.perf_counter_ns()`

Reply:
I'd agree with `time.perf_counter()`, `perf_counter_ns()` could be too much

Reply:
Yeah, I had a read on perf_counter again, and can agree. Single line change should just work.
```suggestion
        self.time_stamp = time.perf_counter()
```

Reply:
I will add a new field so we have this:

```
        self.start_time = time.time()
        self.time_stamp = time.perf_counter()
```

The reason is that `time.perf_counter()` won't give us the start time which we need for mapping power later on.